website/source/docs/providers/aws/r/default_network_acl.html.markdown

```diff
@@ -0,0 +1,151 @@
+---
+layout: "aws"
+page_title: "AWS: aws_default_network_acl"
+sidebar_current: "docs-aws-resource-default-network-acl"
+description: |-
+  Manage the default Network ACL resource.
+---
+
+# aws\_default\_network\_acl
+
+Provides a resource to manage the default AWS Network ACL. VPC Only.
+
+Each VPC created in AWS comes with a Default Network ACL that can be managed, but not
+destroyed. **This is an advanced resource**, and has special caveats to be aware
+of when using it. Please read this document in its entirety before using this
+resource.
+
+The `aws_default_network_acl` behaves differently from normal resources, in that
+Terraform does not _create_ this resource, but instead attempts to "adopt" it
+into management. We can do this because each VPC created has a Default Network
+ACL that cannot be destroyed, and is created with a known set of default rules. 
+
+When Terraform first adopts the Default Network ACL, it **immediately removes all
+rules in the ACL**. It then proceeds to create any rules specified in the 
+configuration. This step is required so that only the rules specified in the 
+configuration are created.
+
+For more information about Network ACLs, see the AWS Documentation on 
+[Network ACLs][aws-network-acls].
+
+## Basic Example Usage, with default rules
```

Comment:
What does "default rules" mean here? Like the equivalent of what AWS gives you?

If that's the case, let's clarify by adding a line like:

> The following config gives the Default Network ACL the same rules that AWS includes, but pulls the resource under management by Terraform. This means that any ACL rules added or changed will be detected as drift.

^^ Double check me - I think that's all true? 😁